src/systems/menu_defn.cpp

```diff
@@ -19,7 +19,7 @@
 
 namespace menu_defn {
 
-static char s_version_str[30];
+static char s_version_str[36];
```

Comment:
?

Reply:
The menu text stored in here is `  Current version: v%s` which is 20 characters and the actual version was `1.1.0-beta1` which is 11 characters, so I presume it was overflowing because there was some junk data being shown in the about page of the menu. I changed it to 36 because get_version_str returns a char[16].

Though this issue would never occur in a non-beta version because the version string would be much shorter without the appended `-beta1`

Reply:
I see, good catch